readthedocs/rtd_tests/fixtures/spec/v2/schema.yml

```diff
@@ -0,0 +1,79 @@
+# Read the Docs configuration file
+
+# The version of the spec to be use
+version: enum('2')
+
+# Formats of the documentation to be built
+# Default: ['htmlzip', 'pdf', 'epub', 'singlehtmllocalmedia']
+formats: any(list(enum('htmlzip', 'pdf', 'epub', 'singlehtmllocalmedia')), enum('all'), required=False)
```

Comment:
We can't break this line, sadly :/

Reply:
I also would remove the `all` option and make it always explicit

Reply:
I think maybe we should even just default to building HTML, and let users specify if they want more. We build a ton of PDF's & Epub's too that are never used.

The original thought was those were good for users, even if the doc owners never used them. Perhaps we could continue building PDF's, but epub's aren't heavily used.

Reply:
> We build a ton of PDF's & Epub's too that are never used

haha, yeah I don't think much people download docs to read offline these days. I think the `all` keyword can remain just to keep consistency around the spec.

Maybe an empty list as default?

Reply:
Most of the people that uses the PDF is because they don't usually have 24/7 internet access --I know people who regularly ask for PDF to be downloaded.

I would like to suggest a _crazy idea_ (for another PR, etc) that could help RTD servers to be cleaner and without tons of useless files: build _extra_ resourses on demand and save a copy (sync servers) for the requested versions only. That way, we will only have the extra resources for those versions that at least where used once.